notebooks/Get Species data from multiple sorces (spocc).qmd

```diff
@@ -0,0 +1,267 @@
+
+Date: February 2025
+
+# Script information
+
+This notebook was developed by the EcoCommons team to support researchers and conservationists in analyzing shorebird distribution data using open data sources. It demonstrates how to query, clean, visualize, and save species occurrence data using the `spocc` package in R.
+
+# Objectives
+
+1.  Overview and Conceptualisation
+2.  Get Data
+3.  Data Cleaning and Filtering
+4.  Data Visualisation
+5.  Save Data
+
+# **Workflow Overview**
+
+Following a standard protocol for species distribution models proposed by Zurell et al., (2020), a R environment set-up step and five main modelling steps are demonstrated in this notebook:
+
++--------------+-----------------------------+------------------------------------------------------------+
+| Step         | Main Step                   | Sub-steps                                                  |
++==============+=============================+============================================================+
+| Set-up       | Set-up                      | S.1 Set up the working directory                           |
+|              |                             |                                                            |
+|              |                             | S.2 Install and load required R packages for this notebook |
++--------------+-----------------------------+------------------------------------------------------------+
+| Step 1       | Conceptualisation           | Taxon, location, scale                                     |
++--------------+-----------------------------+------------------------------------------------------------+
+| Step 2       | Get Data                    | Define and Query Species                                   |
++--------------+-----------------------------+------------------------------------------------------------+
+| Step 3       | Data Cleaning and Filtering | Refine and Check                                           |
++--------------+-----------------------------+------------------------------------------------------------+
+| Step 4       | Data Visualisation          | Map                                                        |
++--------------+-----------------------------+------------------------------------------------------------+
+| Step 5       | Save data                   | Save as CSV and SHP                                        |
++--------------+-----------------------------+------------------------------------------------------------+
+
+In the near future, this material may form part of comprehensive support materials available to EcoCommons users. If you have any corrections or suggestions to improve the efficiency, please [contact the EcoCommons](mailto:support@ecocommons.org.au) team.
+
+![](https://raw.githubusercontent.com/EcoCommons-Australia-2024-2026/ec-notebook_site/main/images/EC_breaker_nobackgoundcolor.png)
```

Comment:
I dont know if this section is really required for this notebook. It made sense for the GLM notebook, but you are not running any SDMs here